cmd/identity/main.go

```diff
@@ -186,12 +186,15 @@ func cmdAuthorize(cmd *cobra.Command, args []string) error {
 	// Ensure we dont enforce a signed Peer Identity
 	config.Signer.TLS.UsePeerCAWhitelist = false
 
-	revDB, err := revocation.NewDBFromCfg(config.Signer.TLS)
+	revocationDB, err := revocation.NewDBFromCfg(config.Signer.TLS)
 	if err != nil {
 		return errs.New("Error creating revocation database: %+v", err)
 	}
+	defer func() {
+		err = errs.Combine(err, revocationDB.Close())
+	}()
 
-	signedChainBytes, err := config.Signer.Sign(ctx, ident, authToken, revDB)
+	signedChainBytes, err := config.Signer.Sign(ctx, ident, authToken, revocationDB)
```

Comment:
I think we could pass nil here. This would only be relevant if the certificates server needed to issue a revocation but I don't think that will ever happen because it's grpc identity is disposable.

Reply:
Let's do a separate PR for that.